apps/web-giddh/src/app/login/login.component.ts

```diff
@@ -491,8 +503,9 @@ export class LoginComponent implements OnInit, OnDestroy {
      * @memberof LoginComponent
      */
     public async appleLogin(): Promise<void> {
+        const whiteLabel = this.generalService.getDecodedWhiteLabel();
         const CLIENT_ID = "com.giddh.appsignin.client"
-        const url = PRODUCTION_ENV || isElectron ? 'https://api.giddh.com' : 'https://apitest.giddh.com';
+        const url = PRODUCTION_ENV || isElectron ? 'https://api.giddh.com' : whiteLabel?.giddhWhiteLabel?.apiDomainName ? `http://${whiteLabel.giddhWhiteLabel.apiDomainName}` : 'https://apitest.giddh.com';
```

Comment:
```suggestion
        const url = PRODUCTION_ENV || isElectron ? 'https://api.giddh.com' : whiteLabel?.giddhWhiteLabel?.apiDomainName ? `http://${whiteLabel.giddhWhiteLabel.apiDomainName}` : 'https://apitest.giddh.com';
```

http:// make this common using constant or any other approch